data/Hydradaboss.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "Olaniyi Oluwaseyi",
+  "bio": " Back-End Engineer ",
+   "tags": [
+    "Open Source",
+    "Javascript",
+     "AWS",
+     "Database",
+    "CI/CD",
+    "Freelancing",
```

```suggestion
    "Freelancing",
```
The trailing comma isnot required